### PR TITLE
Add seizure frequency graph

### PR DESCRIPTION
**Pull request by jmbabazi**

### CODE REVIEW

**Comment by brandones** [comment]
@jmbabazi wanna move it into [GraphFactory](https://github.com/PIH/openmrs-module-mirebalais/blob/master/api/src/main/java/org/openmrs/module/mirebalais/apploader/apps/GraphFactory.java) so it's easy to reuse? See [this example of usage in the CALF](https://github.com/PIH/openmrs-module-mirebalais/blob/54e64256afce94046d52791a5873740476ab72be/api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java#L1344).

**Comment by mogoodrich** [comment]
LGTM, but @brandones thanks for pointing out the GraphFactory you had set up earlier.

@jmbabazi it would be great to move all the Mental Health graphs into this factory method.

File: api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java [hunk 1]
```diff
@@ -2050,18 +2050,8 @@ private void enableEpilepsyProgram() {
...
-                "pih.app.epilepsy.seizureGraph",
-                "icon-bar-chart",
-                null,
-                null,
-                objectNode(
-                        "widget", "obsgraph",
-                        "label", "pih.app.epilepsy.seizureGraph",
-                        "icon", "icon-bar-chart",
-                        "conceptId", MirebalaisConstants.EPI_SEIZURES,
-                        "maxResults", "30"  // MEX-127
```
Comment (brandones) [inline api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java]: MEX-127 is actually unrelated; not sure why it's referenced here. It has to do with Obs Across Encounters.

**Comment by brandones** [comment]
@jmbabazi I made the change. Wanna take a look, and merge it in if it looks good to you?

**Comment by brandones** [comment]
Hey, @jmbabazi , can we close this out or merge it in?

**Comment by jmbabazi** [comment]
Hi @brandones , I think Mark Merged this in already. @mogoodrich , should we close this out?

**Comment by brandones** [comment]
Closing.

**Pull request closed without merging**